diff --git a/Makefile.ffmpeg b/Makefile.ffmpeg
index d405932cb..3d39271fd 100644
--- a/Makefile.ffmpeg
+++ b/Makefile.ffmpeg
@@ -63,11 +63,11 @@ else
 LIBX265_DIFFS  = libx265.pic.diff
 endif
 
-LIBVPX_VER     = 1.7.0
+LIBVPX_VER     = 1.5.0
 LIBVPX         = libvpx-$(LIBVPX_VER)
-LIBVPX_TB      = $(LIBVPX).tar.gz
-LIBVPX_URL     = https://github.com/webmproject/libvpx/archive/v$(LIBVPX_VER)/$(LIBVPX_TB)
-LIBVPX_SHA1    = fb3d4b80596d1e3b1a7f53757d63e7d2b3eeb7c9
+LIBVPX_TB      = v$(LIBVPX_VER).tar.gz
+LIBVPX_URL     = https://github.com/webmproject/libvpx/archive/$(LIBVPX_TB)
+LIBVPX_SHA1    = 82426a59ad34d72c762f9e4079e8e5197afc842d
 
 LIBOGG         = libogg-1.3.3
 LIBOGG_TB      = $(LIBOGG).tar.gz
