diff --git a/Makefile.ffmpeg b/Makefile.ffmpeg
index da9a655..eb1cc14 100644
--- a/Makefile.ffmpeg
+++ b/Makefile.ffmpeg
@@ -52,10 +52,10 @@ LIBX264_URL    = http://ftp.videolan.org/x264/snapshots/$(LIBX264_TB)
 LIBX264_SHA1   = 90c48dab878520a819905880bc4912834fe694fb
 LIBX264_DIFFS  = libx264.configure.diff
 
-LIBX265        = x265_2.1
+LIBX265        = x265_1.9
 LIBX265_TB     = $(LIBX265).tar.gz
 LIBX265_URL    = http://ftp.videolan.org/videolan/x265/$(LIBX265_TB)
-LIBX265_SHA1   = 37ed92b83dbe9e1420e1c226d35caeea16d4b0e4
+LIBX265_SHA1   = 8c9aa3b87b0f0a418bbb9782e9354d112d75e003
 ifeq ($(CONFIG_PIE),yes)
 LIBX265_DIFFS  = libx265.pie.diff
 else
