diff --git a/Makefile.ffmpeg b/Makefile.ffmpeg
index 4036c5002..f7f0581c1 100644
--- a/Makefile.ffmpeg
+++ b/Makefile.ffmpeg
@@ -66,11 +66,11 @@ else
 LIBX265_DIFFS  = libx265.pic.diff
 endif
 
-LIBVPX_VER     = 1.9.0
+LIBVPX_VER     = 1.5.0
 LIBVPX         = libvpx-$(LIBVPX_VER)
-LIBVPX_TB      = $(LIBVPX).tar.gz
-LIBVPX_URL     = https://github.com/webmproject/libvpx/archive/v$(LIBVPX_VER)/$(LIBVPX_TB)
-LIBVPX_SHA1    = 2ab8203ad8922bdf3256e4a197d1348fa8db9a62
+LIBVPX_TB      = v$(LIBVPX_VER).tar.gz
+LIBVPX_URL     = https://github.com/webmproject/libvpx/archive/$(LIBVPX_TB)
+LIBVPX_SHA1    = 82426a59ad34d72c762f9e4079e8e5197afc842d
 
 LIBOGG         = libogg-1.3.4
 LIBOGG_TB      = $(LIBOGG).tar.gz
